index.js

```diff
@@ -105,8 +105,16 @@ function walker(node, parameters, outerFont, customMatcher) {
 
       if (node.name === 'a' && node.attribs && node.attribs.href) {
         parameters.attributes.unshift({
-          type: ns.ATTRIBUTE_LINK,
-          value: node.attribs.href,
+            type: ns.ATTRIBUTE_LINK,
+            value: node.attribs.href,
+            range: [offset, length]
+        }, {
+          type : ns.ATTRIBUTE_FOREGROUND_COLOR,
```

Comment:
Hi Fokke,
I have added an if statement and extra info to the readme that would help users when adding html to the string.xml doc using cdata. I can't seem to get the indexSpec.js to run so I have just taken out what I added I get the error below when running grunt.

~~~
[DEBUG] module.js:327
[DEBUG]     throw err;
[DEBUG]     ^
[ERROR] : Cannot find module '/Users/sam.east/Desktop/ti-html2as/undefined'
[DEBUG]     at Function.Module._resolveFilename (module.js:325:15)
[DEBUG]     at Function.Module._load (module.js:276:25)
[DEBUG]     at Function.Module.runMain (module.js:441:10)
[DEBUG]     at startup (node.js:139:18)
[DEBUG]     at node.js:974:3
[ERROR] Alloy compiler failed
~~~